content/docs/concepts/flops/overview.md

```diff
@@ -0,0 +1,158 @@
+  Flower implements and handles core FL components, but it does not handle many other necessary aspects such as deployment, orchestration, dependency management, containerization, or advanced tracking and observability tools seen in MLOps.
+
+  ### [mlflow](https://mlflow.org/)
+  MLflow is a mature open-source MLOps tool that powers the runtime observability and tracking features of FLOps.
+  MLflow augments and supports the entire ML lifecycle, from conception, to code and dependency archivation and unification, tracking and tagging ML models and artifacts, and re-redeployments. 
+
+  ### Apache's 'Data Suite'
+  FLOps works with real data on learner devices.
+  Several technologies provided by [Apache](https://www.apache.org/) are employed to efficiently receive, store, and retrieve such data.
+  - [Arrow](https://arrow.apache.org/docs/format/Columnar.html) columnar format for working with data in memory.
+  - [Parquet](https://parquet.apache.org/) columnar file format for working with data stored on disk.
+  - [Arrow Flight](https://arrow.apache.org/docs/format/Flight.html) a gRPC-based framework to transport Apache formatted data over the network.
+
+  ### [Buildah](https://buildah.io/)
+  Buildah helps FLOps to build containerized FL images inside running containers.
+  Therefore distributing the computational load of image building across devices.
+
+  ### [Anaconda Suite](https://docs.anaconda.com/)
+  FLOps uses the Anaconda Suite to resolve dependencies to provide equal ML training across devices.
+  To be more specific:
+   - [Miniconda](https://docs.anaconda.com/miniconda/) for lightweight (base) images
+   - [Mamba](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community) for fast dependency resolving
+
+{{< /details >}}
+
+{{< details "Convenient Installation & native CLI" >}}
+  FLOps can be easily set up by cloning its [repository](https://github.com/oakestra/addon-FLOps) and running the docker-compose file on the same machine as the Oakestra Root Orchestrator.
+  See the [FLOps Manuals](/docs/manuals/flops) for concrete instructions.
+
+  The [Oakestra CLI](/docs/getting-started/deploy-app/with-the-cli/#the-oak-cli) supports a [set of commands](/docs/manuals/cli/features/flops-addon/#oak-addon-flops) to work with FLOps.
+{{< /details >}}
+
+{{< details "Designed for Change" >}}
+  FLOps aims to be easily modifiable and extendable by developers and researchers.
+  FLOps employs the following techniques to strive for high code quality and flexibility:
+  - Use state-of-the-art libraries and frameworks
+  - Extendable CLI
+  - Ready-made extendable multi-platform images and services to automate development and evaluation workflows.
+  - Additional base images with optional development flags to speed up the build and execution times.
+  - Enforcement of proper styling and typing via formatters and linters, including CI.
```

Comment:
This is a good summary of the features. Maybe bring this on the top and highlight as a callout?

Reply:
Nice idea - done